components/src/atoms/list-item/ListItem.spec.js

```diff
@@ -0,0 +1,37 @@
+import { mount } from 'vue-test-utils'
+import AListItem from './ListItem.vue'
+
+describe('ListItem', () => {
+  it('has default structure', () => {
+    const wrapper = mount(AListItem)
+
+    expect(wrapper.is('li')).toBe(true)
+    expect(wrapper.classes()).toContain('a-list-item')
+    expect(wrapper.classes().length).toBe(1)
+  })
+
+  it('renders slot text when passed', () => {
+    const wrapper = mount(AListItem, {
+      slots: {
+        default: `
+          <span>List item default text</span>
+        `
+      }
+    })
+
+    expect(wrapper.find('.a-list-item span').exists()).toBe(true)
+    expect(wrapper.find('.a-list-item span').text()).toEqual('List item default text')
```

Comment:
I think you can change selectors to `'.a-list-item > span'` to make sure that `span` is a direct child